src/lib.rs

```diff
@@ -987,6 +987,16 @@ impl Element {
             .collect())
     }
 
+    /// Look up a CSS value for this element by name (color, width, etc)
+    pub async fn css(&mut self, prop: &str) -> Result<Option<String>, error::CmdError> {
+        let cmd = WebDriverCommand::GetCSSValue(self.element.clone(), prop.to_string());
+        match self.client.issue(cmd).await? {
+            Json::String(v) => Ok(Some(v)),
+            Json::Null => Ok(None),
```

Comment:
What exactly does `None` mean here? Does it mean that the element did not exist, that the attribute was not set, or something else? Would be good to add that to the method documentation!

Reply:
Looks like the css command always returns a string, even if the property is not defined. Must have been some issue with my testing previously.... 14a90c8